packages/ra-ui-materialui/src/list/datagrid/Datagrid.tsx

```diff
@@ -317,6 +321,39 @@ export interface DatagridProps<RecordType extends RaRecord = any>
      */
     className?: string;
 
+    /**
+     * The component used to render the bulk actions toolbar.
+     *
+     * @see https://marmelab.com/react-admin/Datagrid.html#bulkactionstoolbar
+     * @example
+     * import { List, Datagrid, BulkActionsToolbar, SelectAllButton, BulkDeleteButton, useUnselectAll, Button } from 'react-admin';
+     *
+     * const UnselectButton = () => {
+     *     const unselect = useUnselectAll('posts');
+     *     return <Button onClick={unselect} label="Unselect all records" />;
+     * };
+     *
+     * const PostBulkActionsToolbar = () => (
+     *     <BulkActionsToolbar actions={
+     *         <>
+     *             <SelectAllButton label="Select all records" />
+     *             <UnselectButton />
```

Comment:
This example doesn't make sense, as we already have an unselect button (the cross). I suggest you create a custom toolbar that doesn't use BulkActionsToolbar, but simply lists the selected ids